source/features/small-user-avatars.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager';
+import getUserAvatarURL from '../github-helpers/get-user-avatar';
+
+function init(): void {
+	for (const link of select.all(
+		':is(.js-issue-row, .js-pinned-issue-list-item) [data-hovercard-type="user"]', // Excludes bots
+	)) {
+		const username = link.textContent!;
+		const size = 16;
+
+		link.prepend(
+			<img
+				className="avatar avatar-user mr-1"
```

Comment:
The avatar has extra space on the left when inside a rgh-collaborator bubble. It should have the same spacing too and bottom (a flexbox might be necessary)

Reply:
I have already added some extra padding. I tried flexbox and it will wreck everything in that line

We don't need perfect aligning when it's that small

Reply:
But this looks *unrefined* 🥲 

![image](https://user-images.githubusercontent.com/1402241/230928107-f2b7ff91-8f65-4982-9fe3-9d468e7fb56a.jpeg)

With inline-flex this shouldn't be excessively difficult to fix. I can look into it tomorrow

Reply:
Currently the vertical padding is smaller

Having it the same as horizontal margin will look like this:

<img width="386" alt="image" src="https://user-images.githubusercontent.com/44045911/230928217-60a44346-e52b-4cb9-abd5-4d6af60605ff.png">

This place is just too small... or you don't mind some extra margin?